plugins/secretdata.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: secretdata
+spec:
+  version: v1.0.1
+  homepage: https://github.com/kei6u/kubectl-secretdata
+  shortDescription: A kubectl plugin for finding decoded secret data with productive search flags
```

Comment:
> A kubectl plugin for

unnecessary

> secret

please capitalize as Secret (consistent with Kubernetes docs).

> finding

is this plugin for "finding" or "viewing"? 

please make sure this field fits 50 chars.